lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py

```diff
@@ -51,22 +51,32 @@
         description:
             - List of dictionaries which specify disks of the virtual machine.
             - A disk can be made accessible via four different types: I(disk), I(lun), I(cdrom), I(floppy).
-            - https://kubevirt.io/api-reference/master/definitions.html#_v1_disk
+            - All possible configuration options are available in U(https://kubevirt.io/api-reference/master/definitions.html#_v1_disk)
         type: list
     volumes:
         description:
             - List of volumes specification for the virtual machine.
-            - https://kubevirt.io/api-reference/master/definitions.html#_v1_volume
+            - All possible configuration options are available in U(https://kubevirt.io/api-reference/master/definitions.html#_v1_volume)
         type: str
     labels:
         description:
             - Labels are key/value pairs that are attached to virtual machines. Labels are intended to be used to
               specify identifying attributes of virtual machines that are meaningful and relevant to users, but do not directly
               imply semantics to the core system. Labels can be used to organize and to select subsets of virtual machines.
               Labels can be attached to virtual machines at creation time and subsequently added and modified at any time.
-            - More on labels that are used for internal implementation:
-              https://kubevirt.io/user-guide/#/misc/annotations_and_labels
+            - More on labels that are used for internal implementation U(https://kubevirt.io/user-guide/#/misc/annotations_and_labels)
         type: dict
+    interfaces:
+        description:
+            - An interface defines a virtual network interface of a virtual machine (also called a frontend).
+            - Each interface must have a corresponding C(network) with the same name.
```

Comment:
Is `same name` required by KubeVirt or is it our simplification?

Reply:
In current approach yes, but if we would go with following design, it wouldnt:

```
kubevirt_vm:
  ...
  interfaces:
    - name: x
       bridge: {}
       network: 
         multus:
           networkName: mynetconf
```

Reply:
Sounds good.

Reply:
Ok, in that case I would do the same also for volumes and disks.